src/frontend/components/videoCard/VideoCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./videoCard.css";
+import cardImage from "../../assets/videoImages/demoImage.webp";
+
+function VideoCard() {
+	return (
+		<div className="video__card">
+			<div className="video__card__badge">
+				<i className="far fa-heart fa-2x"></i>
+			</div>
+			<img alt="" src={cardImage} className="video__card-image" />
```

Comment:
Use a meaningful `alt` for better accessibility

Reply:
done